### ISSUE TITLE
Autocomplete in query does not work for columns with upper case

### ISSUE THREAD

**Issue by jonaskello** [OmniDB/OmniDB#279]
Same context as described in #278.

When I enter this in the query window:

```
select t.*
from dbo."Account" t
where t.Id
```

The auto completion kicks in for the `where t.` part, and correctly suggests the `Id` column, but when I hit enter to select it, it is added without the needed double quotes.

The expected result is this:

```
select t.*
from dbo."Account" t
where t."Id"
```

**Comment by wind39** [comment]
Hi @jonaskello ,

Thanks for reporting this. This bug is already handled and the fix will be included in 2.3, to be released very soon. Stay tuned!

![image](https://user-images.githubusercontent.com/7719686/32180735-eb4976c0-bd79-11e7-8844-71cb49fac50a.png)

![image](https://user-images.githubusercontent.com/7719686/32180809-191c822c-bd7a-11e7-86f6-d9e7d7955a08.png)

**Comment by rafaelthca** [comment]
Fixed in 2.3.0. Closing this.
